azure-pipelines.yml

```
@@ -803,6 +804,17 @@ jobs:
     continueOnError: true
     displayName: test-plume-lib.sh
 
+- job: jspecify_conformance
```

```suggestion
- job: jspecify_conformance_jdk21
```

Also change above. Okay to test with only one version, but follow the same naming convention to make it clear.